internal/venice-common/src/main/java/com/linkedin/venice/pubsub/factory/MetricsParameters.java

```diff
@@ -0,0 +1,24 @@
+package com.linkedin.venice.pubsub.factory;
+
+import io.tehuti.metrics.MetricsRepository;
+
+
+public class MetricsParameters {
+  public final String uniqueName;
```

Comment:
should it be private if not used else where?

Reply:
I just find that we might not need this class. Previously we had it as inner class inside `KafkaClientFactory`, we did not really use it in our code, actually.